pkgs/by-name/da/davis/package.nix

```
@@ -27,8 +29,11 @@ php.buildComposerProject2 (finalAttrs: {
     rm -rf "$out/share"
   '';
 
-  passthru.tests = {
-    inherit (nixosTests) davis;
+  passthru= {
```

```suggestion
  passthru = {
```
Remember to `nixfmt`.